corehq/apps/domain/management/commands/make_superuser.py

```diff
@@ -27,6 +28,9 @@ def get_password_from_user():
 
     @signalcommand
     def handle(self, username, **options):
+        if settings.IS_SAAS_ENVIRONMENT:
+            raise CommandError("""You cannot run this command in SaaS Enviornments.
+            Use https://www.commcarehq.org/hq/admin/superuser_management/ for granting superuser permissions""")
```

Comment:
This URL won't be right for other environments. I think you can get it from `get_site_domain()`.

Reply:
Thanks 👍 
Updated.